internal/saucecloud/cloud.go

```diff
@@ -259,3 +259,18 @@ func (r *CloudRunner) validateTunnel(id string) error {
 	log.Info().Msg("Tunnel is ready!")
 	return nil
 }
+
+func (r *CloudRunner) dryRun(project interface{}, files []string, sauceIgnoreFile string, suiteNames string) error {
+	tmpDir, err := ioutil.TempDir("./", "sauce-app-payload-*")
+	if err != nil {
+		return err
+	}
+	log.Info().Msgf("Detected following test suites: %s.", suiteNames)
+	zipName, err := r.archiveProject(project, tmpDir, files, sauceIgnoreFile)
+	if err != nil {
+		return err
+	}
+
+	log.Info().Msgf("Skipping project upload. Saving bundled project to %s.", zipName)
```

Comment:
It's clear that we are skipping the upload by the virtue of the message not appearing :)
```suggestion
	log.Info().Msgf("Saving bundled project to %s.", zipName)
```